src/olympia/devhub/forms.py

```diff
@@ -582,6 +583,8 @@ class DescribeForm(AkismetSpamCheckFormMixin, AddonFormBase):
     slug = forms.CharField(max_length=30)
     summary = TransField(widget=TransTextarea(attrs={'rows': 4}),
                          max_length=250)
+    description = TransField(widget=TransTextarea(attrs={'rows': 4}),
+                             min_length=10)
```

Comment:
It seems the rows attribute doesn't do anything anyway from a quick test during development - or more likely we override it with css.  I got zero UX guidance on what everything should look like so far so I'm expecting some followup issues.

Reply:
Yeah, no worries here, it's just be blabbering out what's in my mind :)

Reply:
Looking at this more closely, that's actually a bug.

```diff
diff --git a/src/olympia/devhub/forms.py b/src/olympia/devhub/forms.py
index 5029dc0a1d..e439d1e673 100644
--- a/src/olympia/devhub/forms.py
+++ b/src/olympia/devhub/forms.py
@@ -581,9 +581,9 @@ class SourceForm(WithSourceMixin, forms.ModelForm):
 class DescribeForm(AkismetSpamCheckFormMixin, AddonFormBase):
     name = TransField(max_length=50)
     slug = forms.CharField(max_length=30)
-    summary = TransField(widget=TransTextarea(attrs={'rows': 4}),
+    summary = TransField(widget=TransTextarea(widget_attrs={'rows': 5}),
                          max_length=250)
-    description = TransField(widget=TransTextarea(attrs={'rows': 4}),
+    description = TransField(widget=TransTextarea(widget_attrs={'rows': 10}),
                              min_length=10)
     is_experimental = forms.BooleanField(required=False)
     requires_payment = forms.BooleanField(required=False)
diff --git a/src/olympia/translations/tests/test_widgets.py b/src/olympia/translations/tests/test_widgets.py
index d19183914e..566d176658 100644
--- a/src/olympia/translations/tests/test_widgets.py
+++ b/src/olympia/translations/tests/test_widgets.py
@@ -82,3 +82,26 @@ class TestWidget(TestCase):
         actual = widgets.TransInput().value_from_datadict(data, [], 'f')
         expected = {'en-US': 'woo', 'de': 'herr', 'fr': None}
         assert actual == expected
+
+    def test_transtextarea_renders_attrs(self):
+        models.Translation.objects.create(
+            id=666, locale='en-us', localized_string='test value en')
+
+        widget = widgets.TransTextarea(attrs={'rows': 5, 'cols': 20})
+
+        doc = pq(widget.render('foo', 666))
+
+        assert doc('textarea')[0].get('rows') == '5'
+        assert doc('textarea')[0].get('cols') == '20'
+
+    def test_transinput_renders_attrs(self):
+        models.Translation.objects.create(
+            id=666, locale='en-us', localized_string='test value en')
+
+        widget = widgets.TransInput(attrs={'rows': 5, 'cols': 20})
+
+        doc = pq(widget.render('foo', 666))
+
+        print(doc.html())
+        assert doc('input')[0].get('rows') == '5'
+        assert doc('input')[0].get('cols') == '20'
diff --git a/src/olympia/translations/widgets.py b/src/olympia/translations/widgets.py
index f97753118f..af28a5147e 100644
--- a/src/olympia/translations/widgets.py
+++ b/src/olympia/translations/widgets.py
@@ -55,13 +55,14 @@ class TransMulti(forms.widgets.MultiWidget):
         # We do need self.widgets to be non-empty in order for is_hidden to
         # work, so we create a dummy one. It will also serve as fallback when
         # there is no value set already.
-        super(TransMulti, self).__init__(widgets=[self.widget()], attrs=attrs)
+        super(TransMulti, self).__init__(
+            widgets=[self.widget(attrs=attrs)], attrs=attrs)

     def render(self, name, value, attrs=None):
         self.name = name
         value = self.decompress(value)
         if value:
-            self.widgets = [self.widget() for _ in value]
+            self.widgets = [self.widget(attrs=self.attrs) for _ in value]
         else:
             default_locale = getattr(self, 'default_locale',
                                      translation.get_language())
```

Fixes it and allows customization of rows/cols and other `attrs` if we want to.

Imho this looks much better and shows the amount of content that belongs to each field more visually.